MM2RandoLib/Extensions/Int32Extensions.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Text;
+
+namespace MM2Randomizer.Extensions
+{
+    public static class Int32Extensions
+    {
+        public static String ConvertToBase26(this Int32 in_Value)
```

Comment:
I think we should have this whole function operate on unsigned values (UInt32).

Reply:
Random() is initialized with Int32; if the number is negative, it just uses the absolute value as the seed. While I agree in principle, Int32 makes more sense, so we aren't having to do casts and range checks when initializing the seed value.